action.yml

```diff
@@ -1,23 +1,100 @@
-name: 'Setup DDEV in Github Workflows'
-description: |
-  This action installs **DDEV** in your Github Workflow.
+name: 'Setup and start DDEV action'
+description: 'Set up your GitHub Actions workflow with DDEV'
 author: 'Jonas Eberle and DDEV contributors'
-runs:
-  using: 'node20'
-  main: 'lib/main.js'
-branding:
-  icon: cpu
-  color: yellow
+
 inputs:
   ddevDir:
-    description: 'ddev project directory'
+    description: 'Path to your DDEV project. This path needs to contain the .ddev/ directory.'
     required: false
     default: '.'
+
   autostart:
-    description: 'Start ddev automatically'
+    description: 'Starts your DDEV project immediately.'
     required: false
-    default: true
+    default: 'true'
+
   version:
-    description: 'Install a specific ddev version, such as 1.22.4'
+    description: 'Install a specific DDEV version. Example: v1.24.0'
```

Comment:
The version format change is a BC break currently which has to be addressed if this idea gets accepted. Should be easy.